docs/src/ToricVarieties/NormalToricVarieties.md

```diff
@@ -22,118 +22,125 @@ the affine and non-affine case:
 
 ## Constructors
 
+### Affine toric varieties
+
 ```@docs
 AffineNormalToricVariety(C::Cone)
-CyclicQuotientSingularity
 NormalToricVariety(C::Cone)
-NormalToricVariety(PF::PolyhedralFan)
-NormalToricVariety(P::Polyhedron)
-toric_projective_space
-hirzebruch_surface
-del_pezzo
+AffineNormalToricVariety(v::NormalToricVariety)
 ```
 
+### Cyclic Quotient Singularities
 
-## Properties of toric varieties
+Cyclic quotient singularities are quotients of $\mathbb{C}^2$ by the action of
+$\mathbb{Z}/n\mathbb{Z}$ acting via 
+$$\left(\begin{array}{cc}\xi & 0\\0 & \xi^q\end{array}\right)$$,
+where $\xi$ is a $n$-th root of unity, and $0<q<n$ are two coprime integers.
```

Comment:
For me, `$0<q<n$ are two coprime integers` is not clear (as `$0<q<n$` is/are not "two integers"). How about this:
```suggestion
where $\xi$ is a $n$-th root of unity, and $0<q<n$ is coprime with $n$.
```
or even better
```suggestion
where $\xi$ is a $n$-th root of unity, and $q$ is coprime with $n$.
```
(I don't think it is necessary to mention `0<q<n$ for the definition, is it?)

Reply:
This is ok for me. @lkastner, I believe you wrote this code. Are you ok too? Are we overlooking something?

Reply:
This is not ok, as this omits the other two conditions, namely $0<q$ and $q<n$. Is this a formulation issue? The formulation "are coprime integers" and "are two coprime integers" is not that uncommon, but if it makes you happy we can say "coprime with". So I then suggest:
```suggestion
where $\xi$ is a $n$-th root of unity, and $q$ and $n$ are integers, such that $q$ is coprime with $n$, and $0<q<n$.
```

Reply:
Thank you Lars.